client/me/help/help-contact/index.jsx

```diff
@@ -64,6 +65,10 @@ const HelpContact = React.createClass( {
 			this.props.connectHappychat();
 		}
 
+		if ( config.isEnabled( 'help/directly' ) ) {
+			this.props.initializeDirectly();
```

Comment:
Just noting that we chatted about this in Slack and got the :+1: to move forward as-is.